spec/spec.md

```diff
@@ -0,0 +1,601 @@
+
+[[ref:VCs]] in Web5 are cryptographically secure, privacy-respecting, and machine-verifiable digital credentials. They enable the assertion and verification of claims about subjects in a decentralized manner. Key features of VCs in Web5 include:
+
+1. **Cryptographic Security**: VCs are digitally signed, ensuring their integrity and authenticity.
+2. **Privacy Preservation**: VCs enable user-control with features like selective disclosure.
+3. **Standardization**: VCs follow standardized data models, promoting interoperability across industries.
+4. **Flexibility**: VCs can represent a wide range of credentials and attributes.
+5. **Verifiability**: Claims in VCs can be independently verified by relying parties.
+
+::: note
+Future versions of this specification may adopt other models of [[ref:verifiable credentials]], such as [ISO mDLs](https://www.iso.org/standard/69084.html), [SD-JWT-based Verifiable Credentials](https://datatracker.ietf.org/doc/draft-ietf-oauth-sd-jwt-vc/), or [W3C Verifiable Credentials v2.0](https://www.w3.org/TR/vc-data-model-2.0/).
+:::
+
+#### Supported Credential Types
+
+Web5 primarily supports the W3C Verifiable Credentials Data Model v1.1 [[ref:VC-DATA-MODEL-1.1]], with the following features:
+
+1. **Basic Credentials**: Standard Verifiable Credentials containing claims about a subject.
+2. **[[ref:Verifiable Presentations]]**: Holder signed VCs that can be shared with verifiers.
+3. **JSON Schema**: VCs that use [[ref:VC-JSON-SCHEMA]] for claim validation.
+4. **Status List Credentials**: VCs that support revocation and suspension using [[ref:STATUS-LIST-2021]].
+
+#### VC Features
+
+Conformant [[ref:Web5 SDKs]] ****MUST**** support the following [[ref:Verifiable Credential]] operations:
+
+1. **Issuance**: Create and sign [[ref:Verifiable Credentials]] using the `vc-jwt` format.
+2. **Verification**: Verify the integrity, authenticity, and status of [[ref:Verifiable Credentials]] as `vc-jwt`.
+3. **Presentation Creation**: Generate [[ref:Verifiable Presentations]] from one or more [[ref:VCs]] as `vp-jwt`.
+4. **Presentation Verification**: Verify the integrity and authenticity of [[ref:Verifiable Presentations]] as `vp-jwt`.
+5. **Status**: Manage the status of issued credentials using [[ref:STATUS-LIST-2021]].
+
+:::todo
+[](https://github.com/TBD54566975/web5-spec/issues/12) Specify credential validation in more detail.
+:::
+
+#### VC Formats
+
+Web5 supports the following credential formats:
+
+1. **JWT**: Verifiable Credentials and Presentations secured as JSON Web Tokens (JWTs) [[spec:RFC7519]] using the `vc-jwt` and `vp-jwt` types.
+2. **JSON-LD**: Support for [[ref:JSON-LD]] context in VCs, although [[ref:JSON-LD]] processing is not performed.
+
+#### Credential Status
+
+Conformant [[ref:Web5 SDKs]] ****MUST**** support Credential Status as specified by [[ref:STATUS-LIST-2021]] for usage with the W3C Verifiable Credential Data Model 1.1 [[ref:VC-DATA-MODEL-1.1]].
```

Comment:
not using the updated one? https://www.w3.org/TR/vc-bitstring-status-list/

Reply:
there are some weird things that will break if we do, mostly incompatibility with the 1.1 data model since bitstring is designed to be used with 2.0